### PR TITLE
ArrayModelField Subnested - fix

### PR DESCRIPTION
**Pull request by lesantana09**
This is a test with ArrayModelField Subnested, correction by @klausthelen merge and tests by @lesantana09, this is a important fix by use Subnested Array in ArrayModelField djongo

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -91,9 +91,9 @@ def find_version(*file_paths):
         'Driver for allowing Django to use MongoDB as the database backend.'),
     install_requires=[
         'sqlparse==0.2.4',
-        'pymongo>=3.2.0',
+        'pymongo==3.8.0',
```
Comment (nesdis) [inline setup.py]: Why is this == 3.8, instead of >=3.8?

File: setup.py [hunk 2]
```diff
@@ -91,9 +91,9 @@ def find_version(*file_paths):
...
         'django>=2.0',
-        'dataclasses>=0.1',
+        'dataclasses==0.6',
```
Comment (nesdis) [inline setup.py]: Again why the == requirement

File: djongo/models/fields.py [hunk 3]
```diff
@@ -117,14 +117,14 @@ class DictField(Field):
     def get_db_prep_value(self, value, connection, prepared=False):
         if not isinstance(value, dict):
             raise ValueError(
-                f'Value: {value} must be of type dict'
+                f'Value: {value} must be of type list'
```
Comment (nesdis) [inline djongo/models/fields.py]:
```suggestion
                f'Value: {value} must be of type dict'
```
Why exactly has this been changed?

File: setup.py (hunk 2 above)
Comment (lesantana09) [reply setup.py]: Hi @nesdis. This is only fix to tests, I'm changed again

File: setup.py (hunk 1 above)
Comment (lesantana09) [reply setup.py]: Hi @nesdis. This is only fix to tests, I'm changed again

**Comment by nesdis** [comment]
Thanks for the commit

**Pull request merged**